doc/md/guides/mysql/cd-101.md

```diff
@@ -0,0 +1,156 @@
+---
+id: mysql-cd-101
+title: Detecting drops of foreign-key constraints on MySQL
+slug: /guides/mysql/checks/CD101
+tags: [linting, ci, destructive-change, mysql, drop-foreign-key]
+---
+
+## Introduction
+
+This document describes how Atlas's linter can be used to warn about dropping
+foreign key constraints from a MySQL database.
+
+Let's demonstrate by creating a database with 2 tables - users and pets:
```

Comment:
This introduction feels too short. See Rotem's article as an example: https://atlasgo.io/guides/mysql/checks/DS103#introduction